docs/migration_guide.rst

```diff
@@ -61,4 +62,12 @@ Interface of :ref:`Contract` remains unchanged and it is still the recommended w
 AccountClient
 -------------
 
-:ref:`AccountClient` now implements ``Client`` ABC: parameters of some of its' methods has changed.
\ No newline at end of file
+:ref:`AccountClient` now implements ``Client`` ABC: parameters of some of its' methods has changed.
```

Comment:
now implements ``Client`` **interface**: parameters of some of its methods **were** changed.